website/src/docs/react.md

```diff
@@ -31,48 +27,55 @@ Instead of adding a UI plugin to an Uppy instance with `.use()`, the Uppy instan
 All other props are passed as options to the plugin.
 
 ```js
-import React from 'react'
+import React, { useEffect } from 'react'
 import Uppy from '@uppy/core'
-import Tus from '@uppy/tus'
-import { DragDrop } from '@uppy/react'
-
-const uppy = new Uppy({
-  meta: { type: 'avatar' },
-  restrictions: { maxNumberOfFiles: 1 },
-  autoProceed: true,
-})
-
-uppy.use(Tus, { endpoint: '/upload' })
-
-uppy.on('complete', (result) => {
-  const url = result.successful[0].uploadURL
-  store.dispatch({
-    type: 'SET_USER_AVATAR_URL',
-    payload: { url },
-  })
-})
-
-const AvatarPicker = ({ currentAvatar }) => {
-  return (
-    <div>
-      <img src={currentAvatar} alt="Current Avatar" />
-      <DragDrop
-        uppy={uppy}
-        locale={{
-          strings: {
-            // Text to show on the droppable area.
-            // `%{browse}` is replaced with a link that opens the system file selection dialog.
-            dropHereOr: 'Drop here or %{browse}',
-            // Used as the label for the link that opens the system file selection dialog.
-            browse: 'browse',
-          },
-        }}
-      />
-    </div>
-  )
+import { Dashboard } from '@uppy/react'
+
+const uppy = new Uppy()
+
+function Component () {
+  useEffect(() => {
+    return () => {
+      uppy.close({ reason: 'unmount' })
+    }
+  }, [])
+
+  return <Dashboard uppy={uppy} />
 }
 ```
 
+### Dynamic options
+
+```js
+import React, { useState, useEffect } from 'react'
+import Uppy from '@uppy/core'
+import { Dashboard } from '@uppy/react'
```

Comment:
Do we want to highlight `import Dashboard from '@uppy/react/lib/Dashboard.js'`? We probably want to introduce an exports map to sugar code it, but importing the individual modules makes the most sense otherwise you have to fight with peer dependencies.

Reply:
I was hoping to do a separate PR for the exports map next to this so we don't have to document that workaround